src/subscription/SubscriptionSelector.js

```diff
@@ -39,10 +40,10 @@ class _SubscriptionSelector {
 				m(BuyOptionBox, this._createProUpgradeBoxAttr(vnode.attrs))
 			]
 		} else {
-			// Shows different order of BuyBoxes if screen with is smaller than 560 px.
-			// Changes order of BuyBoxes to Premium Pro Free, needed for mobile view
-			const minWidthTwoColumns = size.first_col_min_width + size.second_col_min_width + 20;
-			if (window.innerWidth < minWidthTwoColumns ) {
+			// Add BuyOptionBox margin twice to the boxWidth received
+			const columnWidth = vnode.attrs.boxWidth + 20;
```

Comment:
I think that we should reference a common variable where the `BuyOptionBox`margin is defined.
I have tried to use a static variable within the `BuyOptionBox` class, but I get a flow error, and exporting directly a variable from the file where `BuyOptionBox` is defined doesn't seem to be a common practice in the repo. What would you suggest?

Reply:
You can use static variables like here
https://github.com/tutao/tutanota/blob/c9b63f9c159127268bab24853000e37456058460/src/gui/base/Dialog.js#L58

but the best would probably be just
```js
export const BOX_MARGIN = 10
```

Reply:
At first I have tried exactly the solution suggested by you, but from [BuyOptionsBox](https://github.com/tutao/tutanota/blob/c9b63f9c159127268bab24853000e37456058460/src/subscription/BuyOptionBox.js#L111), the class itself is not exported, but it exports the type `Class<MComponent<BuyOptionBoxAttr>>`, and it would haven't had that static variable (from here the flow error).
Anyway, I opted for the second solution, thanks :-)

Reply:
Ah, I get it now, it's an artifact, I just didn't get to change all those places yet where it re-exports class constant instead of doing `export class A implements MComponent<Attrs>`. It's fine with exporting standalone constant for now.